### DISCUSSION TITLE
http_request_failed cURL error 60

### DISCUSSION THREAD

**Discussion by kostritsaalex** [leokhoa/laragon#917] [Q&A]
Hi guys, 

Can you help me to address this issue:
`	Array
(
    [0] => http_request_failed cURL error 60: SSL certificate problem: self signed certificate
)
	`

What was done:
1) Followed that guideline: https://pen-y-fan.github.io/2022/11/10/how-to-set-up-laragon-on-a-new-windows-computer-part-4/
2) Downloaded 'cacert.pem' and Updated php.ini conf additionally:

`[curl]
; A default value for the CURLOPT_CAINFO option. This is required to be an
; absolute path.
;curl.cainfo =
curl.cainfo = "C:\laragon\bin\php\php-8.1.10-Win32-vs16-x64\extras\ssl\cacert.pem"

[openssl]
; The location of a Certificate Authority (CA) file on the local filesystem
; to use when verifying the identity of SSL/TLS peers. Most users should
; not specify a value for this directive as PHP will attempt to use the
; OS-managed cert stores in its absence. If specified, this value may still
; be overridden on a per-stream basis via the "cafile" SSL stream context
; option.
openssl.cafile = "C:\laragon\bin\php\php-8.1.10-Win32-vs16-x64\extras\ssl\cacert.pem"`

Thanks!

**Comment by Astriel** [comment]
Hey @kostritsaalex I'm facing the exact same issue, everything was working well until yesterday, and now I'm facing this error too.

**Comment by eltharin** [comment]
Hi, check if your antivirus has not https protection (HTTPS Scanning on norton)  activated

**Comment by rescenic** [comment]
Don't open any browsers. Apache & NGINX option > SSL > Add Laragon.crt to Trust Store. Then open the browser you choose.
